plugins/module_utils/common.py

```diff
@@ -181,8 +216,16 @@ def auth_set(*names):
         client = get_api_client._pool[digest]
         return client
 
+    def generate_cache_file(kubeclient):
+        import hashlib
```

Comment:
Let us import this with rest of the imports.

Reply:
Moved where all the imports are done.